src/utils/pathUtils.ts

```diff
@@ -29,20 +32,24 @@ export const normalizePath = (path: string) =>
  * @param _basePath - path, which is probably parent
  * @param _path - path, which is probably child of _basePath
  */
-export const isInnerPath = (_basePath: string, _path: string) => {
+export const isInnerPath = (_basePath: string | symbol, _path: string | symbol) => {
+    if (_basePath === ROOT_PATH || _path === ROOT_PATH) return true;
```

Comment:
This line of code don't have tests, please run `npm run test:coverage`, and check all the places, which are not covered. 
Also, this is not correct logic, because function should check, is `_path` argument inner path of `_basePath`. If `_basePath` is ROOT_PATH, then any provided path will be inner, but it should not work vice versa - ROOT_PATH is not inner path of any path.

Reply:
Left only file useObservers not covered 
![image](https://user-images.githubusercontent.com/64651944/103481624-dec0ed80-4de4-11eb-885a-04d06dcfe104.png)